zerver/lib/notifications.py

```diff
@@ -511,12 +511,13 @@ def enqueue_welcome_emails(user: UserProfile) -> None:
     context = common_context(user)
     context.update({
         'unsubscribe_link': unsubscribe_link,
-        'organization_setup_advice_link':
-        user.realm.uri + '/help/getting-your-organization-started-with-zulip',
-        'getting_started_with_zulip_link':
-        user.realm.uri + '/help/getting-started-with-zulip',
+        'user_role_group': 'admins' if user.is_realm_admin else 'users',
+        'getting_started_link':
+            user.realm.uri +
+            ('/help/getting-your-organization-started-with-zulip'
+             if user.is_realm_admin else
+             '/help/getting-started-with-zulip'),
```

Comment:
I refactored this to use new if statements; the result is a lot more readable.  Also, you wanted to have tagged "admins" and "users" for translation.

Reply:
See b4084d30d29bf935e90812502af132374e0cdbc2.  As a general rule, I don't think the `if ... else ...` all in one line is a good feature; it decreases readability, and also the nice thing about doing proper if statements is that then our test coverage tooling will tell us whether a case is totally untested.

Reply:
Good to know, thanks for the tip!